### ISSUE TITLE
Problem with option "every"

### ISSUE THREAD

**Issue by tidakdijual** [katzer/cordova-plugin-local-notifications#1104]
I got a problem when I used "every" on my option. the notification won't come if I set it to minute (I don't know if it also affect others option like hour, day or so).

But it fixed by option At (A with uppercase), but when I try use at, it's not working.

this an example code:

```
var alarmTime = new Date();
cordova.plugins.notification.local.schedule({
  id: 10,
  title: "test notif!",
  text: "content of notification",
  //at: alarmTime, //this one won't work
  At: alarmTime, //this one is working
  every: "minute",
  data: { meetingId:"1" }
})
```

**Comment by payousefi** [comment]
Hey @tidakdijual -- I just ran into this issue as well, and think I've figured it out. Are you seeing any notifications at all? Or is only the first showing up?

In my case it seems the every 'minute' interval only works 1 minute after the previous or first notification is dismissed. I was expecting the notifications to pile up per minute, but it actually allows for only one at a time to stay on the notifications list (which seems alright to me, but I wasn't expecting it originally). Thought I'd share what I noticed in case you were still wondering about this!
